ecc/bls12381/ff/fp12.go

```diff
@@ -72,6 +68,22 @@ func (z *Fp12) ExpVarTime(x *Fp12, n []byte) {
 	*z = *zz
 }
 
+func (z *Fp12) UnmarshalBinary(b []byte) error {
+	return errFirst(
+		z[1].UnmarshalBinary(b[:Fp6Size]),
+		z[0].UnmarshalBinary(b[Fp6Size:2*Fp6Size]),
+	)
+}
+
+func (z Fp12) MarshalBinary() (b []byte, e error) {
+	if b1, e := z[1].MarshalBinary(); e == nil {
```

Comment:
You don't return the error because `:=` create a second instance of `e` that shows the return value.

Reply:
fixed now